src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/mgmt_iot_hub/swagger_iot_hub_resource_provider.json

```diff
@@ -0,0 +1,1913 @@
+        }
+      }
+    },
+    "IotHubProperties": {
+      "description": "The Iot Hub properties.",
+      "type": "object",
+      "properties": {
+        "state": {
+          "description": "IoT Hub state.",
+          "type": "string"
+        },
+        "provisioningState": {
+          "description": "Provision state.",
+          "type": "string"
+        },
+        "authorizationPolicies": {
+          "description": "The authorization rules.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SharedAccessSignatureAuthorizationRule"
+          }
+        },
+        "hostName": {
+          "description": "The name of the host.",
+          "type": "string"
+        },
+        "eventHubEndpoints": {
+          "description": "The event hub endpoint properties.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/EventHubProperties"
+          }
+        },
+        "storageEndpoints": {
+          "description": "The list of storage end points where files can be uploaded. Currently only one storage account can be configured.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/StorageEndpointProperties"
+          }
+        },
+        "messagingEndpoints": {
+          "description": "The list of messaging end points configured.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/MessagingEndpointProperties"
+          }
+        },
+        "enableFileUploadNotifications": {
+          "description": "The flag which indicates whether file upload notification should be enabled. This is optional at iot hub level. When enabled upload notifications will be available.",
+          "type": "boolean"
+        },
+        "cloudToDevice": {
+          "$ref": "#/definitions/CloudToDeviceProperties"
+        },
+        "comments": {
+          "description": "The comments.",
+          "type": "string"
+        },
+        "operationsMonitoringProperties": {
+          "$ref": "#/definitions/OperationsMonitoringProperties"
+        },
+        "features": {
+          "description": "The Capabilities/Features that need to be enabled for the Hub.",
+          "enum": [
+            "None",
+            "DeviceManagement"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "Capabilities",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "IotHubSkuInfo": {
+      "description": "The Sku related information for the hub.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the Sku.",
+          "enum": [
+            "F1",
+            "S1",
+            "S2",
+            "S3"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "IotHubSku",
+            "modelAsString": true
```

Comment:
>modelAsString": true [](start = 13, length = 20)

@johanste what's the guidance on modelAsString: true? In other packages we have treated the parameters differently based on this value (this would imply that sku have a completer rather than a choice list, for example)